operator/v1/types_network.go

```diff
@@ -328,6 +331,16 @@ type HybridOverlayConfig struct {
 	HybridOverlayVXLANPort *uint32 `json:"hybridOverlayVXLANPort,omitempty"`
 }
 
+type HardwareOffloadConfig struct {
+	// enableHardwareOffload specifies whether or not OVS hardware offload
+	// should be enabled. If unset, this property defaults to 'false' and
+	// OVS hardware offload is disabled.
+	EnableHardwareOffload *bool `json:"enableHardwareOffload"`
```

Comment:
What happens if this is `true` but `Policy = "none"`? Will hardware offload work / do anything? Can we get away with a single flag?

Reply:
Yes, if hw-offload=true && tc-policy=none,  packets will first be processed by hardware and be directed to ovs software datapath only when hardware miss.

This flag (EnableHardwareOffload) is designed to be equivalent to setting `ovs-vsctl set Open_vSwitch . other_config:hw-offload=true`. However, it's not restrictly required as we can use tc-policy (HardwareOffloadPolicy) to deduce whether hw-offload shall be enabled or disabled. For example :if tc-policy is configured to one of 'none', 'skip_sw' or 'skip_hw', EnableHardwareOffload will be set to true.

Reply:
@squeed would you prefer to removing the `EnableHardwareOffload` ?

Reply:
If Enabled and Policy = None makes sense, then it's OK to have both fields. Just making sure we don't add extra fields we don't need.

Reply:
right, tc-policy is only relevant when hw-offload is enabled.

Reply:
If you default to false, why you need a pointer here?

Reply:
@soltysh which is more k8s-conventional: setting `Spec.HardwareOffload = nil` disables offload, or adding an explicit `Spec.HardwareOfflload.Enabled = false`? I feel like I knew but not I'm not so sure.

Reply:
In other words, maybe we can get rid of `EnableHardwareOffload` entirely, and just have the nil struct be sufficient.

Reply:
Empty `HardwareOffload` struct works for me as well. I'm just saying we don't need boolean pointers w/o explicit reason.

Reply:
sure sure, just asking about the convention.

Reply:
Updated to use bool value instead of pointer.